content/information-distribution/source.md

```diff
@@ -75,6 +77,63 @@ In such an event, client implementations **should** consider articles posted
 prior to the key being declared as compromised as probably not having been
 tampered with, with no way to be 100% sure about it.
 
+## Change of Matrix user
+
+A source or sub-source might have to change the Matrix user it uses to publish
+information at some points in its life time. This can happen for many reason:
+the homeserver it was registered on went down, or got compromised, or got
+isolated from the rest of the federation...
+
+In such an event, the source or sub-source **must** publish a new registration
+event from its new Matrix user with the `prev_id` key having its previous user's
+ID as its value, and the `prev_event` key having the ID of the latest event
+emitted prior to the change as its value.
+
+This key **must** be used by client implementations to link the previous user to
+the new one, considering both as the same entity. A missing `prev_id` key or an
+empty string simply means that the source or sub-source didn't previously use
+another Matrix account to publish information. In this case, the `prev_event`
+key can be omitted as well.
+
+If the entity is a source (and not a sub-source), trust authorities certifying
+it as trustworthy **must** update their list of trusted entities by removing the
+previous user from it, and adding the new one to it. The signature associated
+with the new user **must** be generated from the new user's registration event.
+
+Client implementations **should** define a threshold for linking a source to its
+new user. This threshold is defined by the percentage of trust authorities
+certifying this source as trustworthy that updated their list of trusted sources
+to reference the new user, within the trust authorities trusted by the user. If
+a threshold is defined by a client implementation, it **should** consider the
+link between a source and its new user as valid once that percentage becomes
+higher than the threshold. Such a threshold isn't necessary for sub-sources,
+because a sub-source is only verified by its parent source, which then **must**
+update its list of sub-sources by removing the sub-source's previous user and
+adding the new one.
+
+Once the link between a source (or a sub-source) and its new user is consider
+valid, client implementations **must** consider both the source's (or
+sub-source's) new user and its previous one (and older ones if the source or
+sub-source changed its user more than once) as the same entity. This means that
+articles published by the source's (or the sub-source's) previous user **must**
```

Comment:
`This means that
articles published by the source's (or the sub-source's) previous user before the event defined in prev_event **must**`